packages/react-components/README.md

```diff
@@ -32,10 +32,3 @@ ReactDOM.render(
 ### Docs
 
 Docs are hosted at https://aka.ms/fluentui-storybook.
-
-Publishing docs:
-
-1. Set `CHROMATIC_PROJECT_TOKEN` in your shell
-1. Run `yarn chromatic`
-
-> @levithomason and @miroslavstastny currently have the token for alpha stage publishing.
```

Comment:
removing the section as it is not useful for public audience, we will find a better place for it with @miroslavstastny